artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java

```diff
@@ -1671,7 +1671,7 @@ private void parseClusterConnectionConfiguration(final Element e, final Configur
    private void parseGroupingHandlerConfiguration(final Element node, final Configuration mainConfiguration) {
       String name = node.getAttribute("name");
       String type = getString(node, "type", null, Validators.NOT_NULL_OR_EMPTY);
-      String address = getString(node, "address", null, Validators.NOT_NULL_OR_EMPTY);
+      String address = getString(node, "address", "", Validators.NO_CHECK);
```

Comment:
why "" for default not Null now you're removing the NOT NULL OR EMPTY check?

Reply:
I did that to match the change that was made for the cluster-connection address parsing.  See https://github.com/apache/activemq-artemis/blob/master/artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java#L1581.